src/envo/envo-edit.owl

```diff
@@ -40171,6 +39843,535 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3400002> "ar
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_3510002> "ATF")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510002> "automatic transmission fluid"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510002> <http://purl.obolibrary.org/obo/ENVO_00003074>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510004> (hydraulic fluid)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Hydraulic_fluid") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_3510004> "A medium by which power is transferred in hydraulic machinery.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_3510004> <http://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_3510004> "2020-08-03T17:35:45Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_3510004> "http://purl.enanomapper.org/onto/ENM_8000199")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_3510004> "Common hydraulic fluids are based on mineral oil or water.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510004> "hydraulic fluid"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510004> <http://purl.obolibrary.org/obo/ENVO_00003074>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510005> (motor oil)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Motor_oil") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_3510005> "A lubricant for internal combustion engines which 1) reduces friction and wear 2) cleans the engine 3) neutralizes acids 4) improves sealing of piston rings, and 5) cools the engine.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_3510005> <http://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_3510005> "2020-08-03T17:37:56Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_3510005> "engine lubricant")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_3510005> "engine oil")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510005> "motor oil"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510005> <http://purl.obolibrary.org/obo/ENVO_00003074>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510006> (diesel fuel)
```

Comment:
Good asserted positioning, but I'd add that subclass axiom to have this inferred into manufactured product.